src/components/KakaoLogin/AdlipLogin.js

```diff
@@ -33,8 +36,12 @@ function AdlipLogIn() {
     })
       .then(res => res.json())
       .then(res => {
-        if (res.token) {
-          localStorage.setItem('token', res.token);
+        const adlipToken = res.token;
+        const userSocialPlatform = res.socialPlatform;
```

Comment:
이렇게 비구조화, 이름 변경 할 수 있습니다 

```suggestion
        const { token: adlipToken, socialPlatform: userSocialPlatform } = res;
```